package.json

```diff
@@ -11,22 +11,22 @@
     "@webassemblyjs/helper-module-context": "1.9.0",
     "@webassemblyjs/wasm-edit": "1.9.0",
     "@webassemblyjs/wasm-parser": "1.9.0",
-    "acorn": "^7.3.0",
+    "acorn": "^7.4.0",
     "chrome-trace-event": "^1.0.2",
     "enhanced-resolve": "5.0.0-beta.10",
-    "eslint-scope": "^5.0.0",
+    "eslint-scope": "^5.1.0",
     "events": "^3.0.0",
     "glob-to-regexp": "^0.4.1",
-    "graceful-fs": "^4.1.15",
+    "graceful-fs": "^4.2.4",
     "json-parse-better-errors": "^1.0.2",
     "loader-runner": "^4.0.0",
-    "mime-types": "^2.1.26",
-    "neo-async": "^2.6.1",
+    "mime-types": "^2.1.27",
+    "neo-async": "^2.6.2",
     "pkg-dir": "^4.2.0",
-    "schema-utils": "^2.5.0",
-    "tapable": "^2.0.0-beta.10",
-    "terser-webpack-plugin": "^3.0.2",
-    "watchpack": "2.0.0-beta.14",
+    "schema-utils": "^2.7.0",
+    "tapable": "^2.0.0-beta.11",
+    "terser-webpack-plugin": "^3.1.0",
+    "watchpack": "^2.0.0-beta.14",
```

Comment:
No need `^`, it is better update dep by step, not together, some updates require rewrite tests

Reply:
Okay. Maybe I will try to update step by step, and see which updates don't require any rewriting.